src/components/navigation/document-tab-panel.tsx

```diff
@@ -182,7 +183,7 @@ export class DocumentTabPanel extends BaseComponent<IProps, IState> {
             );
           })
         }
-        {user.isNetworkedTeacher &&
+        {(user.isNetworkedTeacher && showNetworkDocuments) &&
```

Comment:
Seems like it might be cleaner to move the `user.isNetworkedTeacher` test up into the parent. That way we don't have half the test happening in the parent and half happening here and instead just say that it's the parent's responsibility to determine when to show network documents.

Reply:
Fixed.